src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -17,9 +17,21 @@
 
     public IntakeMap getIntakeMap();
 
+    public interface ShooterMap {
+        public SendableSpeedController shooterWheel();
```

Comment:
Mark this as `default` and then make it return a new `MockSpeedController`. That way you don't have to reimplement it in every derived interface if it's just a mock one.

Reply:
Do this in pretty much every method that returns an SSC